### PR TITLE
Feature/pablo

### PR DESCRIPTION
**Pull request by Sample134567**
Add allure reporting for cypress and update the makefile

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -104,6 +104,5 @@ dist
 .tern-port
 */screenshots/
 *cypress/videos/
-
-#vscode
-.vscode/
\ No newline at end of file
+*allure-report
```
Comment (artkamote) [inline .gitignore:107]: Why did you remove my commit?
.vscode/

File: package.json [hunk 2]
```diff
@@ -20,6 +26,10 @@
     "prettier": "^2.7.1"
   },
   "devDependencies": {
-    "cypress": "^10.3.0"
+    "@shelex/cypress-allure-plugin": "^2.28.0",
+    "cypress": "^10.3.0",
+    "mochawesome": "^7.1.3",
+    "mochawesome-merge": "^4.2.1",
+    "mochawesome-report-generator": "^6.2.0"
   }
```
Comment (artkamote) [inline package.json:31]: remove all unused package like mochawesome*

File: .gitignore (hunk 1 above)
Comment (Sample134567) [reply .gitignore:107]: Will Back this one sorry

File: package.json (hunk 2 above)
Comment (Sample134567) [reply package.json:31]: Will remove

**Review by artkamote — APPROVED** [review]
